.vscode/settings.json

```diff
@@ -9,14 +9,22 @@
     "${workspaceRoot}/setup.cfg"
   ],
   "cSpell.words": [
-    "FACILITYUST",
+    "agrc",
     "aprx",
     "arcpy",
+    "DATASETNAME",
     "facilityid",
+    "FACILITYUST",
```

Comment:
should this be FacilityPST since we're removing FacilityUST?

Reply:
this is only a spelling dictionary so it doesn't really matter